src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form_search form_big"
+    action="#"
+    data-qa="big"
+    >
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      class="form_search form_big"
      action="#"
      data-qa="big"
    >
```

Reply:
Thank you)
It's done